src/pages/EditMonitor.vue

```diff
@@ -179,6 +179,16 @@
                                 </i18n-t>
                                 <br>
 
+                                <label for="jsonPathOperator" class="form-label">{{ $t("Json Query Operator") }}</label>
```

Comment:
As stated in https://github.com/louislam/uptime-kuma/issues/4614#issuecomment-2016835547, please include this as a inside the expected value field instead. I don't think adding another full configratuion option is nessesary for this.

=> here:
![Screenshot from 2024-03-25 17-01-03](https://github.com/louislam/uptime-kuma/assets/26258709/1b410757-eaf1-4b60-bf7b-b9143cb70492)

Reply:
I've played around with different options but none are an option as they don't look great.

*Select with operator in front* - Text completely misaligned 
![grafik](https://github.com/louislam/uptime-kuma/assets/18144723/da2fc125-d586-4e3f-8aa4-ee317761ab70)

*Formatting* 
Within the `<select>` tab, formatting is not possible

*VueMultiselect*
Not really an option in combination with the requirement for having the operator field in the same row as ther expected value as the dropdown can only use the width that its parent has. Looks sub-optimal.

Do you have any style guidelines / best practices available to orient on?

Reply:
> Do you have any style guidelines / best practices available to orient on?

Currently, we don't have a case where the element is a selection button, only buttons and modals.
I think in this case 

![image](https://github.com/louislam/uptime-kuma/assets/26258709/1612707b-6962-40ca-913d-3a8f4d03fed2)

I think something like this is possible via dropdown buttons:
![image](https://github.com/louislam/uptime-kuma/assets/26258709/b5ae08a7-afa3-4618-aef2-75f01353847c)
https://getbootstrap.com/docs/5.2/forms/input-group/#buttons-with-dropdowns